Mathlib/Data/Int/Bitwise.lean

```diff
@@ -270,16 +264,7 @@ theorem bitwise_xor : bitwise xor = Int.xor := by
     <;> simp only [bitwise, natBitwise, Bool.not_false, Bool.bne_eq_xor,
       cond_false, cond_true, negSucc.injEq, Bool.false_xor,
       Bool.true_xor, Bool.not_true,
-      Int.xor, HXor.hXor, Xor.xor, Nat.xor]
-  · congr
-    funext x y
-    cases x <;> cases y <;> rfl
-  · congr
-    funext x y
-    cases x <;> cases y <;> rfl
-  · congr
-    funext x y
-    cases x <;> cases y <;> rfl
+      Int.xor, HXor.hXor, Xor.xor, Nat.xor] <;> simp
```

Comment:
Can you combine these calls?

Reply:
The change I wanted you to try was replacing `simp only [X, Y, Z] ; simp` by `simp [X, Y, Z]`: if the latter works, I consider it preferable.